.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
```

Comment:
The whole file can probably be removed because we can't start a VM inside Travis.
ref. https://stackoverflow.com/questions/37307114/launching-a-vagrant-vm-inside-travis-ci